### PR TITLE
feature: did client call addTrack/addStream

### PR DESCRIPTION
**Pull request by fippo**

### CODE REVIEW

File: features.js [hunk 1]
```diff
@@ -1618,6 +1618,26 @@ module.exports = {
         return feature;
     },
 
+    calledAddStream: function(client) {
+        for (var i = 0; i < peerConnectionLog.length; i++) {
```
Comment (ggarber) [inline features.js]: Can't these be one liners?
```
return !!peerConnectionLog.find(log => log.type === 'addStream');
```

File: features.sql [hunk 2]
```diff
@@ -297,5 +297,7 @@ CREATE TABLE features_import (
     firstremotestreamaudio character varying(255),
     firstremotestreamvideo character varying(255),
     usingsimulcast boolean,
-    sdpSemantics: character varying(32)
+    sdpSemantics: character varying(32),
+    calledAddtrack: boolean,
```
Comment (ggarber) [inline features.sql]: it looks like the previous names are all lowercase "firstremotestreamaudio".  Maybe you want to maintain that style.

File: features.js (hunk 1 above)
Comment (fippo) [reply features.js]: need to port the whole thing to es6...

File: features.sql (hunk 2 above)
Comment (fippo) [reply features.sql]: redshift doesn't care afaik. Not sure about postgres... @fritzy?

Comment (fippo) [reply features.sql]: it does, thanks @argggh

File: features.js (hunk 1 above)
Comment (fippo) [reply features.js]: hrm. missing peerConnectionLog :-(

**Pull request merged**